src/head/script.js

```diff
@@ -154,7 +154,7 @@ export default (config, globalOneTrustEvent) => {
 				if (config.enableGA && !optout && !piiOptout) {
 					OneTrust.InsertHtml('', 'head', insertGoogleAnalytics, null, 'C0004');
 				}
-				if (config.enableGTM && !optout) {
+				if (config.enableGTM && !optout && !piiOptout) {
```

Comment:
@mcstover @emuvente I was curious, do we definitely want to exclude GTM when we opt out of GA or are there use cases I'm not thinking of? The trigger for these changes was that I noticed GTM scripts being loaded on UI pages when testing the cookie changes on dev when the PII opt out was enabled only (new requirement for opting out of signals).

Reply:
GTM is loading things besides GA. I thought the PII opt out was only for GA itself?

Reply:
I was curious though since we're also checking `!optout` here from the opt-out cookie. Should we also not check the new PII output requirement for signals?